Runtime/CurrentChildChangedEventArgs.cs

```diff
@@ -2,6 +2,7 @@
 
 namespace Innoactive.Creator.Core
 {
+    [Obsolete("This event is not used anymore.")]
```

Comment:
Is there are alternative to use?

Reply:
Not really, it has not been used in the whole v1.x.
I am not even sure when or why was it used.